third_party/terraform/tests/resource_compute_instance_from_template_test.go

```diff
@@ -159,7 +159,6 @@ func TestAccComputeInstanceFromTemplate_012_removableFields(t *testing.T) {
 					testAccCheckComputeInstanceExists(resourceName, &instance),
 
 					// Check that fields were able to be removed
-					resource.TestCheckResourceAttr(resourceName, "service_account.#", "0"),
```

Comment:
This is expected to fail because there's always a service account. When unset, it's the default one.